concordium-node/src/p2p/connectivity.rs

```diff
@@ -362,24 +411,28 @@ pub fn accept(node: &Arc<P2PNode>) -> Fallible<Option<Token>> {
             && candidates_lock.len() + conn_read_lock.len()
                 >= node.config.hard_connection_limit as usize
         {
-            bail!("Too many connections, rejecting attempt from {}", addr);
+            return Err(AcceptFailureReason::TooManyConnections);
```

Comment:
It could be helpful to log the source address, since we're dropping this connection.

Reply:
Done.